src/circuit/tx.rs

```diff
@@ -22,6 +24,10 @@ pub struct CTransferPub<C:CS> {
     pub out_commit: CNum<C>,
     pub delta: CNum<C>, // int64 token delta, int64 energy delta, uint32 blocknumber
     pub memo: CNum<C>,
+    pub day: CBoundedNum<C, { DAY_SIZE_BITS }>,
+    pub daily_limit: CBoundedNum<C, { TURNOVER_SIZE_BITS }>,

```

Comment:
If the `daily_limit` input will be zero, nobody will be able to interact with the pool in any way, right? Effectively, the pool will become frozen, correct?

Reply:
Yes, it should work in such a way.